x-pack/plugins/enterprise_search/public/applications/app_search/components/engine/types.ts

```diff
@@ -38,6 +51,7 @@ export interface EngineDetails extends Engine {
   isMeta: boolean;
   engine_count?: number;
   includedEngines?: EngineDetails[];
+  search_relevance_suggestions: SearchRelevanceSuggestions;
```

Comment:
Not sure if its worth getting the BE team to switch this and `last_updated` to use camelCase.